<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DbConnectDialog</class>
 <widget class="QDialog" name="DbConnectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>192</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inventory</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/Icons/icons/Contact.png</normaloff>:/Icons/icons/Contact.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="hostname_label">
         <property name="text">
          <string>Host name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="database_label">
         <property name="text">
          <string>Database:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="user_label">
         <property name="text">
          <string>User name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="password_label">
         <property name="text">
          <string>Password:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="host_lineEdit"/>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="database_lineEdit"/>
         </item>
         <item>
          <widget class="QToolButton" name="database_toolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="user_lineEdit"/>
       </item>
       <item>
        <widget class="QLineEdit" name="password_lineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="remember_checkBox">
     <property name="text">
      <string>Remember me on this computer</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="error_label">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>Error!</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="open_toolButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::TabFocus</enum>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
       <property name="shortcut">
        <string notr="true">Enter, Return</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::DelayedPopup</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="cancel_toolButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>open_toolButton</sender>
   <signal>clicked()</signal>
   <receiver>DbConnectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancel_toolButton</sender>
   <signal>clicked()</signal>
   <receiver>DbConnectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>124</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
